src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -153,7 +157,20 @@ def _get_helm_version(ignore_errors):
 
     if warning:
         print_warning(warning)
-    print("Helm version:\n{}".format(output))
+
+    # Retrieve the helm version if regex pattern is found
+    match_obj = HELM_VERSION_REGEX.search(output)
+    if match_obj:
+        output = match_obj.group(1)
+
+    print("Helm version: {}".format(output))
+
+    # Display an error message if the current helm version < min required version
+    if match_obj and LooseVersion(output) < LooseVersion(MIN_HELM_VERSION):
```

Comment:
What's the difference between `LooseVersion` and `StrictVersion` ?

Reply:
From what I've been able to understand, `StrictVersion` raises exception if the software versioning is not appropriate or doesn't follow standard format. Strings such as a993.#@1.332 would raise exceptions.

If the versioning system follows industry standards, i.e.  majorVersion.minorVersion.Patch, `StrictVersion` and `LooseVersion` will have similar behavior.

In the CLI codebase, `LooseVersion` has been used throughout for comparing software versions. I didn't find any uses of `StrictVersion`. Therefore, I felt that `LooseVersion` would be more appropriate for our use case.

Reply:
I see, makes sense. Good to know. Thanks!